packages/contracts/src/previous/ERC20Token/ERC20Token_v1.sol

```diff
@@ -1,8 +1,8 @@
 pragma solidity ^0.4.11;
 
-import "./Token.sol";
+import { Token_v1 as Token } from "../Token/Token_v1.sol";
 
-contract StandardToken is Token {
+contract ERC20Token_v1 is Token {
```

Comment:
This does not look right. What happened over here?

Reply:
I just renamed it for consistency, seems much more descriptive.